owncloudApp/src/main/java/com/owncloud/android/db/UploadResult.java

```diff
@@ -29,7 +29,7 @@
     FOLDER_ERROR(3),
     CONFLICT_ERROR(4),
     FILE_ERROR(5),
-    PRIVILEDGES_ERROR(6),
+    PRIVILEGES_ERROR(6),
```

Comment:
api break?

Reply:
It should not break anything. If you rename it everywhere, it will be fine. We store the value in the database, not the name. 👍